test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10529,11 +10533,11 @@ def test_quantized_conv2d(self):
         model = torch.nn.quantized.Conv2d(16, 33, 3, stride=2)
         # Manually initialize model weight and bias to random numbers.
         # By default all zeros.
-        q_weight = torch.quantize_per_tensor(torch.randn(33, 16, 3, 3), 0.2, 0, torch.qint8)
-        bias = torch.randn(33)
+        q_weight = torch.quantize_per_tensor(torch.randn(33, 16, 3, 3), 0.5, 0, torch.qint8)
```

Comment:
It is along the general rule of picking scale and zeropoint to avoid overflow. That is commented on top of the quantized tests.
The scale is altered here to accommodate bias which is changed to non-zeros.